static/js/upload_widget.js

```diff
@@ -152,7 +152,7 @@ var upload_widget = (function () {
                     input_error.show();
                     clear();
                 } else {
-                    accept(file);
+                    accept();
```

Comment:
This change is wrong and I think reflects a bug in your static checker; we define a function "accept" earlier in this scope that takes an argument.